src/components/resource/AdminMenu.jsx

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+import React from 'react/addons';
```

Comment:
if you `import React, {PropTypes} from 'react';`, you can use `PropTypes` directly, which makes for less verbose code.